fontData.js

```diff
@@ -12,6 +12,76 @@ const buildAxes = font => {
   }));
 };
 
+const guessFontStyle = fontName => {
+  const styles = [
+    "italic",
+    "thin",
+    "hairline",
+    "extra-light",
+    "extra light",
+    "extralight",
+    "ultra-light",
+    "ultra light",
+    "ultralight",
+    "light",
+    "normal",
+    "regular",
+    "medium",
+    "semi-bold",
+    "semi bold",
+    "semibold",
+    "demi-bold",
+    "demi bold",
+    "demibold",
+    "bold",
+    "extra-bold",
+    "extra bold",
+    "extrabold",
+    "ultra-bold",
+    "ultra bold",
+    "ultrabold",
+    "black",
+    "heavy",
+    "extra-black",
+    "extra black",
+    "extrablack",
+    "ultra-black",
+    "ultra black",
+    "ultrablack",
+    "ultra-condensed",
+    "ultra condensed",
+    "ultracondensed",
+    "extra-condensed",
+    "extra condensed",
+    "extracondensed",
+    "condensed",
+    "semi-condensed",
+    "semi condensed",
+    "semicondensed",
+    "normal",
+    "semi-expanded",
+    "semi expanded",
+    "semiexpanded",
+    "expanded",
+    "extra-expanded",
+    "extra expanded",
+    "extraexpanded",
+    "ultra-expanded",
+    "ultra expanded",
+    "ultraexpanded"
+  ];
+
+  const matches = styles.filter(style =>
+    fontName.toLowerCase().includes(style) ? style : false
```

Comment:
`Array.filter` returns all items for which the function returns a 'truthy' value, so this can be simplified to:

```
const matches = styles.filter(style =>
    fontName.toLowerCase().includes(style));
```

Reply:
What can I say, I have a knack for overcomplicating things 😆 Thanks, great suggestion!